docs/how-to/run-local.md

```diff
@@ -0,0 +1,15 @@
+If you have a requirement to run on your local machine or you are unable to get it to work on
+Docker, do the following steps.  WARNING: If you run into issues you will get limited support.
+
+Run these commands from the app directory:
+1. From the terminal:
+```
+cd app
+cp .env.local-example .env.local
+```
+2. Inspect .env.local and change values as appropriate.
+3. From the terminal in the app directory, run:
+```
+source ../scripts/setscriptpath.sh
+source start-local.sh
```

Comment:
When I ran this I got an error message about not finding the file. I checked that the script path is in $PATH
![Screenshot 2024-01-02 230643](https://github.com/hackforla/peopledepot/assets/1160105/be4190ce-fa82-479e-add4-aecc09e57520)

Reply:
If I ran `../scripts/start-local.sh` it complains about loadenv.sh not found.
![Screenshot 2024-01-02 231640](https://github.com/hackforla/peopledepot/assets/1160105/1fbb6422-c94a-4310-be73-373402d9bf72)